### ISSUE TITLE
How to reclaim funds in on chain lnd wallet. 

### ISSUE THREAD

**Issue by jetlagv1** [lightningnetwork/lnd#2506]
### Background

I had a lnd node running on a raspberry pi with raspiblitz. It worked perfect for quite a while. I started tinkering and f-ed it up. So I did a reinstall with the seed and password. The node is up and running again and the seed worked since it has the same adress etc. The funds however are gone. I understand that the restoring of channels is still work in progress so I consider them lost for now but there was more in the on chain wallet then in the channels but the on chain wallet stays empty as well. 

### Your environment

Latest version of raspiblitz. Fully synced and connected. Shows up in 1ML under the same adress but empty. 

### Steps to reproduce

Tell us how to reproduce this issue. Please provide stacktraces and links to code in question.

### Expected behaviour

I would expect my on chain BTC to show up in the lnd wallet after restoring with seed + password. 

### Actual behaviour

All balances show 0

**Comment by molxyz** [comment]
@jetlagv1 If you have the data files in directory .lnd (linux) you can try by running it on another machine, preferably not on a rpi, a laptop can work, just to see if you can bring back up your node and channels and balances...

**Comment by molxyz** [comment]
@jetlagv1 Yes. It should run fine and make sure the bitcoind is fully synced.

**Comment by Roasbeef** [comment]
Ensure that you have the correct seed. In the logs, do you see it scanning blocks to recover the addresses? You may also need to specify a large look ahead parameter which can be done in the interactive "create" CLI dialogue.
